### PR TITLE
maintainers: remove andreabedini

### PR DESCRIPTION
**Pull request by andreabedini**
###### Motivation for this change

I haven't used Nix or NixOS in a while. <3 :wave:

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lovesegfault** [comment]
Can you remove yourself from the packages you maintain as well?
```
pkgs/development/python-modules/aws-sam-translator/default.nix
33:    maintainers = [ lib.maintainers.andreabedini ];

pkgs/development/tools/aws-sam-cli/default.nix
99:    maintainers = with maintainers; [ andreabedini lo1tuma ];
```

**Comment by andreabedini** [comment]
@lovesegfault thanks for the quick reply. I just removed myself from the two packages. There are not other maintainers for `aws-sam-translator` is that going to be a problem? Thanks

**Comment by lovesegfault** [comment]
Commit messages need to adhere to a common style. They would be:
```
aws-sam-cli: remove andreabedini from maintainers
aws-sam-translator: andreabedini from maintainers
maintainers: remove andreabedini
```

The order here is important too, since you can only remove yourself from the maintainer list once you are no longer listed in any pkgs.

> There are not other maintainers for aws-sam-translator is that going to be a problem?

Not really, all that means is that if it ever breaks and no one wants to deal with the breakage the package will be removed. Not an issue :)

File: pkgs/development/python-modules/aws-sam-translator/default.nix [hunk 1]
```diff
@@ -30,6 +30,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/awslabs/serverless-application-model";
     description = "Python library to transform SAM templates into AWS CloudFormation templates";
     license = lib.licenses.asl20;
-    maintainers = [ lib.maintainers.andreabedini ];
+    maintainers = [ ];
```
Comment (lovesegfault) [inline pkgs/development/python-modules/aws-sam-translator/default.nix:33]: Just delete this line instead of leaving it as an empty list

**Comment by lovesegfault** [comment]
Cheers!

File: pkgs/development/python-modules/aws-sam-translator/default.nix [hunk 2]
```diff
@@ -30,6 +30,5 @@ buildPythonPackage rec {
     homepage = "https://github.com/awslabs/serverless-application-model";
     description = "Python library to transform SAM templates into AWS CloudFormation templates";
     license = lib.licenses.asl20;
-    maintainers = [ lib.maintainers.andreabedini ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/aws-sam-translator/default.nix:33]: In the future please add an empty maintainer field to make it clear that this package is unmaintained.

**Pull request merged**